orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/support/StageData.groovy

```diff
@@ -58,6 +63,10 @@ class StageData {
     return account ?: credentials
   }
 
+  long getDelayBeforeCleanup() {
```

Comment:
Not tied to names but wanted something reasonably consistent and we've got a few eager souls that have already started using `delayBeforeDisableSec`.